src/utils/Overlay/ConnectedOverlay/ConnectedOverlayContent.tsx

```diff
@@ -149,21 +149,11 @@ export const ConnectedOverlayContent: FC<
       onEscape
     });
 
-    useEffect(() => {
-      if (refs.reference && refs.reference.current && overlayIndex) {
-        (refs.reference.current as HTMLElement).style?.setProperty(
-          'zIndex',
-          overlayIndex.toString()
-        );
-      }
-      // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [refs.reference.current, overlayIndex]);
-
     return (
       <OverlayPortal
         id={id}
         ref={refs.setFloating}
-        style={floatingStyles}
+        style={{ ...floatingStyles, 'z-index': overlayIndex }}
```

Comment:
should be this:

```suggestion
        style={{ ...floatingStyles, zIndex: overlayIndex }}
```

Reply:
It's wrong, I was my first try and did't work
https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty

From docs:
![Screenshot 2024-06-13 at 15 44 32](https://github.com/reaviz/reablocks/assets/5399389/c3725567-e5e1-4048-ab1b-b38d519e89ab)